### PR TITLE
fixes cqc working outside chef assigned areas

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 /mob/living/carbon/human/proc/check_block()
 	if(mind)
-		if(mind.martial_art && prob(mind.martial_art.block_chance) && in_throw_mode && !stat && !IsKnockdown() && !IsStun())
+		if(mind.martial_art && prob(mind.martial_art.block_chance) && mind.martial_art.can_use(src) && in_throw_mode && !stat && !IsKnockdown() && !IsStun())
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human_defense.dm]: wait, looking at this, does this not bother to check if someone's restrained?

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -44,7 +44,7 @@
 			return spec_return
 
 	if(mind)
-		if(mind.martial_art && mind.martial_art.deflection_chance) //Some martial arts users can deflect projectiles!
+		if(mind.martial_art && mind.martial_art.can_use(src) && mind.martial_art.deflection_chance) //Some martial arts users can deflect projectiles!
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human_defense.dm]: same here

File: code/modules/mob/living/carbon/human/human_defense.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/mob/living/carbon/human/human_defense.dm]: Should I add it?

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 3]
```diff
@@ -44,7 +44,7 @@
 			return spec_return
 
 	if(mind)
-		if(mind.martial_art && mind.martial_art.deflection_chance) //Some martial arts users can deflect projectiles!
+		if(mind.martial_art && !incapacitated(FALSE, TRUE), mind.martial_art.can_use(src) && mind.martial_art.deflection_chance) //Some martial arts users can deflect projectiles!
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/human_defense.dm]: `,`

**Pull request merged**